### DISCUSSION TITLE
can not upload large size source in Resources

### DISCUSSION THREAD

**Discussion by gcnote** [apache/dolphinscheduler#13725] [General]
dolphin scheduler version: 3.1.3
deploy model: Standalone

I need to upload jar to hdfs, and the jar is more than 1G,  unfortunately, the upload is failed.
Is there any solution to fix this?

**Comment by Tandoy** [comment]
related: https://github.com/apache/dolphinscheduler/issues/10340   https://github.com/apache/dolphinscheduler/issues/10509

**Comment by gcnote** [comment]
您好，您发的邮件我已收到，谢谢！！！
